src/main/kotlin/com/project/twiliospring/config/TwilioConfiguration.kt

```diff
@@ -16,4 +19,11 @@ class BeanConfiguration(
     fun initTwilio() {
         Twilio.init(twilioSID, twilioToken)
     }
+
+    @Profile("twilio")
```

Comment:
why do we need profile here?

Reply:
since this service was intended to be used in only one service, it was moved to this service